joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/MixedAstCreationPassTest.scala

```diff
@@ -1264,4 +1264,20 @@ class MixedAstCreationPassTest extends AbstractPassTest {
     }
   }
 
+  "AST generation for import/require" should {
```

Comment:
There is a test suite somewhere already testing many import / dependency related stuff. Could you put this there? E.g., https://github.com/joernio/joern/blob/master/joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/DependencyAstCreationPassTest.scala